repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/lks/newscategories.yaml

```diff
@@ -0,0 +1,29 @@
+/content/documents/administration/valuelists/lks/newscategories:
```

Comment:
Could we get rid of this value-list for now as it's just a sample/dummy value-list ?

Reply:
Done.